src/main/java/org/eyeseetea/sdk/common/ExternalAppConstants.java

```diff
@@ -0,0 +1,31 @@
+package org.eyeseetea.sdk.common;
+
+
+import static org.eyeseetea.sdk.common.RequiredChecker.required;
+
+import android.content.Context;
+
+public class ExternalAppConstants {
```

Comment:
this class is dangerous because DatabaseUtils depended on it and can throw NullPointerException if ExternalAppConstants is not initialized previously. I think is better remove this class and pass context and database name as parameters to DatabaseUtils class when is needed